src/python/gennylib/auto_tasks.py

```diff
@@ -345,7 +345,14 @@ def all_tasks_modern(tasks: List[GeneratedTask]) -> Configuration:
 
             t = c.task(task.name)
             t.priority(5)
-            t.commands([CommandDefinition().function("f_run_dsi_workload").vars(bootstrap)])
+            t.commands(
+                [
+                    CommandDefinition()
+                    .command("timeout.update")
+                    .params({"exec_timeout_secs": 86400, "timeout_secs": 7200}),  # 24 hours
```

Comment:
How did you come up with these values?

Reply:
This is what it was before I killed all the "legacy" logic.